recipe/activate.bat

```diff
@@ -0,0 +1,9 @@
+if defined CONDA_BUILD_STATE (
+    @echo on
+)
+
+if defined CUDA_PATH (
+    set "CONDA_CUPY_CUDA_PATH=%CUDA_PATH%"
+)
+
+set "CUDA_PATH=%CONDA_PREFIX%"
```

Comment:
My understanding is that this code always set CUDA_PATH environment variable in conda environment where cupy is installed. I think's it's better to avoid this, because

* This will make users impossible to use their own CUDA installation outside of conda (e.g. for example users who want to use `backend=nvcc` will need it because `cudatoolkit-dev` is unavailable for Windows)
* This may affect behavior of other libraries.

Reply:
@kmaehashi This has been set in the Linux builds (`{de}activate.sh`) for quite some time, and the fact that we haven't received any bug report seems to suggest it's safe 🙂 Given this empirical test in the wild, I intend to follow the same practice on Windows and see how it goes. 

There's a very long discussion for why we need to set `CUDA_PATH` here, see https://github.com/cupy/cupy/issues/3222#issuecomment-605307182 but also the whole issue. I remember one of the motivations is that there were some bugs related to how CuPy looks up the CUDA libraries. Without setting this, there will be random failures due to CuPy mistakenly looking up, say, `/usr/local/cuda` when it should not.

Just FYI, Conda ensures the packages are linked to the `cudatoolkit` components installed in the same Conda env. It is not possible to link to local CUDA installations, so to do this users have to install the wheel that you distribute. However, if users have a local CUDA installation version-matched to the installed `cudatoolkit`, there is no problem running `backend='nvcc'`. I made sure of it 🙂

`cudatoolkit-dev` should not exist in the first place, as it seriously violates the CUDA EULA. I don't know why it has not been taken down. But it's a no-go solution on any platform.

Reply:
Thanks, sorry I was not catching up with the discussion in #3222, will take a look later. I think it's fine to proceed with the release if no one complained so far 🙂

> Just FYI, Conda ensures the packages are linked to the cudatoolkit components installed in the same Conda env. It is not possible to link to local CUDA installations, so to do this users have to install the wheel that you distribute. However, if users have a local CUDA installation version-matched to the installed cudatoolkit, there is no problem running backend='nvcc'. I made sure of it 🙂

Ah, that sounds right if `nvcc` is on `PATH`.

Reply:
No worries, that issue was almost 1 year old, and things have evolved quite a bit since then! Always happy to discuss and reconsider the decisions made before 👍